Gemfile.lock

```diff
@@ -325,7 +327,7 @@ GEM
       nokogiri (~> 1.8)
 
 PLATFORMS
-  ruby
+  x86_64-darwin-19
```

Comment:
Is this important?

Reply:
Any time `bundle install` is run on a new type of machine, it gets added to the PLATFORMS section.  `x86_64-darwin-19` is a MacOS platform.  OK to leave it there.